myriadeploy/create_deployment.py

```diff
@@ -129,8 +138,7 @@ def main(argv):
                            '(default is [--name])')
 
     parser.add_argument(
-        '--heap', type=str, help='Java VM heap size (e.g., "-Xmx2g") '
-                                 'and/or other parameters')
+        '--heap', type=float, help='Java VM heap size in GB (e.g., "-Xmx2g")')
```

Comment:
This parameter should be changed to something more descriptive, e.g., `--max-heap-size-gb`. Also the sample usage is now wrong (the parenthesized part should just be deleted).